addon/utils/element/estimate-element-height.js

```diff
@@ -1,8 +1,8 @@
-import { assert } from 'smoke-and-mirrors/-debug/helpers';
+import Ember from 'ember';
 
 export default function estimateElementHeight(element, fallbackHeight) {
-  assert(`You called estimateElement height without a fallbackHeight`, fallbackHeight);
-  assert(`You called estimateElementHeight without an element`, element);
```

Comment:
It looks like `babel-plugin-filter-imports` [doesn't actually remove the imports](https://github.com/ember-cli/babel-plugin-filter-imports#example) - which seems odd to me, but am sure there's a good reason. You'd have to leave the `-debug` files in the prod/test builds if you want to import them from files that are reachable in prod/test code.